app/controllers/applications_controller.rb

```diff
@@ -0,0 +1,51 @@
+class ApplicationsController < ApplicationController
+
+  def show
+    @application = Application.find(params[:id])
+    @pets = @application.pets
```

Comment:
Do you need this here? I don't see anywhere in your view where you rely on `@pets` instance variable.